pkg/master/master.go

```diff
@@ -259,7 +258,7 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget,
 		batchrest.RESTStorageProvider{},
 		certificatesrest.RESTStorageProvider{},
 		// TODO(enisoc): Remove crdRESTOptionsGetter input argument when TPR code is removed.
```

Comment:
> Are you going to remove crdRESTOptionsGetter in a later PR?

Yes.  I think the rest of changes ripple out to code we're keeping, so they'll need more careful review.